hadoop-hdds/docs/content/tools/Debug.md

```diff
@@ -0,0 +1,477 @@
+---
+title: "Ozone Debug"
+date: 2024-10-14
+summary: Ozone Debug command can be used for all the debugging related tasks.
+---
+<!---
+  Licensed to the Apache Software Foundation (ASF) under one or more
+  contributor license agreements.  See the NOTICE file distributed with
+  this work for additional information regarding copyright ownership.
+  The ASF licenses this file to You under the Apache License, Version 2.0
+  (the "License"); you may not use this file except in compliance with
+  the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+Ozone Debug command (`ozone debug`) is a collection of developer tools intended to help in debugging and get more information of various components of ozone.
+
+```bash
+$ ozone debug --help 
+Usage: ozone debug [-hV] [--verbose] [-conf=<configurationPath>]
+                   [-D=<String=String>]... [COMMAND]
+Developer tools for Ozone Debug operations
+      -conf=<configurationPath>
```

Comment:
Indent of --conf is different than others.

Reply:
It is aligned with the beginning of `--set` in the below line, as there is no one-letter short-form for it.